src/Build/Graph/GraphBuilder.cs

```diff
@@ -257,8 +259,7 @@ private static void AddEdgesFromSolution(IReadOnlyDictionary<ConfigurationMetada
                         string.Join(";", entryPoints.Select(e => e.ProjectFile))));
             }
 
-            ErrorUtilities.VerifyThrowArgument(entryPoints.Count == 1, "StaticGraphAcceptsSingleSolutionEntryPoint");
```

Comment:
Note: removed this as the exact same check is done above already. Also, this error message requires a param, which this is not providing, so even if this code were hit it'd result in incorrect behavior.